index.html

```diff
@@ -0,0 +1,85 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <div class="container">
+            <div class="second-container">
+                <div>My Logo</div>
+                <nav>
+                    <svg width="18" height="12" viewBox="0 0 18 12" fill="none" xmlns="http://www.w3.org/2000/svg">
+                        <path fill-rule="evenodd" clip-rule="evenodd" d="M0 1C0 0.447715 0.447715 0 1 0H17C17.5523 0 18 0.447715 18 1C18 1.55228 17.5523 2 17 2H1C0.447715 2 0 1.55228 0 1ZM0 6C0 5.44772 0.447715 5 1 5L17 5C17.5523 5 18 5.44772 18 6C18 6.55229 17.5523 7 17 7L1 7C0.447715 7 0 6.55228 0 6ZM1 10C0.447715 10 0 10.4477 0 11C0 11.5523 0.447715 12 1 12H17C17.5523 12 18 11.5523 18 11C18 10.4477 17.5523 10 17 10H1Z" fill="#6070FF"/>
+                    </svg>
+                </nav>
+            </div>
+        </div>
+    </header>
+
+    <section>
+        <div></div>
+        <div class="section-head">
+            <p>I'm Echezona</p>
+            <p>Glad to see</p>
+            <p>you!</p>
+        </div>
+        <div class="section-body">
+            <p>I’m a software developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        </div>
+        <p>LET'S CONNECT</p>
+        <div class="icons">
```

Comment:
- [ ] Let' connect should be a link `a` or `button` tag, and follow the button styling: Active, Hover and Pressed 

## Figma design
![crr8](https://user-images.githubusercontent.com/103386183/198583033-f73a7897-ca13-4eee-98c0-ef7c3ae82dee.png)


> 
Apart from being semantically correct, the `a` tag will let you connect to other sections of the website, using href.